MODULE_ACCEPTANCE_CRITERIA.MD

```diff
@@ -20,7 +20,7 @@ Please see [Before Development](MODULE_EVALUATION_TEMPLATE#before-development) f
 1. Module adheres to Community Code of Conduct
 1. Module license is compatible with the FOLIO Project
 1. Module can be included in existing community build processes
-1. Module has robust testing that can run with existing community testing processes
+1. Module has robust and fully automated testing procedure supplied and documented by the development team
```

Comment:
How can the development team provide such a testing procedure?

(I thought this group's remit was to review static code analysis, rather than automated testing)

Reply:
No. 80% code coverage. "Fully automated" is supposed to mean that the tests are automatically carried out upon each Pull Request.

Reply:
i.e. that they are not "manual".